src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -49,8 +49,8 @@
 from psyclone.psyir.frontend.fparser2 import Fparser2Reader
 
 
-@pytest.fixture(scope="function")
-def fort_writer():
+@pytest.fixture(scope="function", name="fort_writer")
+def fixture_fort_writer():
```

Comment:
That's nicer - thanks.

Reply:
Yes, I was glad when I learned that this can be done - much better than dozens of pylint directives.